pkg/volume/glusterfs/glusterfs.go

```diff
@@ -516,7 +516,7 @@ func (plugin *glusterfsPlugin) collectGids(className string, gidTable *MinMaxAll
 
 		gid, err := convertGid(gidStr)
 		if err != nil {
-			glog.Error(err)
+			glog.Errorf("failed to parse gid(%s): %v", gidStr, err)
```

Comment:
The pattern we have in this source file is `gid[%s]` , ie all other error message use `[]` instead of `()`, so please keep same format for consistency.